package.json

```diff
@@ -22,9 +22,10 @@
   "homepage": "https://github.com/qgrid/ng#readme",
   "dependencies": {
     "angular": "^1.5.8",
-    "angular-messages": "^1.5.8",
     "angular-material": "^1.1.4",
-    "angular-sanitize": "^1.5.8"
+    "angular-messages": "^1.5.8",
+    "angular-sanitize": "^1.5.8",
+    "ng-expression-builder": "^1.0.0"
```

Comment:
can u put it to dev dependencies? it will be a core plugin that will be included in the source code itself